doc/weakly_supervised.rst

```diff
@@ -700,6 +700,60 @@ of triplets that have the right predicted ordering.
 Algorithms
 ----------
 
+.. _scml:
+
+:py:class:`SCML <metric_learn.SCML>`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sparse Compositional Metric Learning
+(:py:class:`SCML <metric_learn.SCML>`)
+
+`SCML` learns an squared mahalanobis distance from triplet constraints by 
+optimizing sparse positive weights assigned to a set of :math:`K` locally discriminative 
```

Comment:
remove "locally discriminative", which refers to LDA bases (the basis set can be arbitrary)

Reply:
(this applies also to the docstring somewhere I think)